lib/drinks_router.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var bodyparser = require('body-parser');
+var fs = require('fs');
+var path = require('path');
+var counter = 0;
+
+module.exports = function (router) {
+  router.use(bodyparser.json());
+  console.log('hello');
+
+  router.get('/drinks', function (req, res) {
+    fs.readdir('./data', function (err, files) {
+      if (err) {
+        console.log(err);
+        return res.status(500).json({msg: 'server error'});
+      }
+      res.json(files);
+    })
+  });
+
+  router.post('/drinks', function (req, res) {
+    counter++;
```

Comment:
This route should have a checker to make sure that the file does not already exist.